client/src/graphics/animated_sprite.lua

```diff
@@ -37,12 +36,6 @@ end
 AnimatedSprite =
 class(
   function(self, animations, switchFunc)
-    --Signal.turnIntoEmitter(self)
-    --self:createSignal("animSwitched")
-    --self:createSignal("animEnded")
-    --self:createSignal("animLooped")
-    --self:createSignal("animStarted")
-
```

Comment:
Was commented out but as a paradigm, imo no other code should react to animation state ever. Only game state.
Aside from that just cleaning up unused stuff.